gtsfm/loader/astronet_loader.py

```diff
@@ -253,3 +262,21 @@ def is_valid_pair(self, idx1: int, idx2: int) -> bool:
             validation result.
         """
         return super().is_valid_pair(idx1, idx2) and abs(idx1 - idx2) <= self._max_frame_lookahead
+
+
+def get_nonzero_intensity_mask(img: Image, eps: int = 5, kernel: Tuple[int, int] = (15, 15)) -> np.ndarray:
```

Comment:
I think the argument name should be `kernel_size`, not `kernel`

Reply:
changed